test/functional/search_api_test.rb

```diff
@@ -26,7 +50,67 @@ def app
     assert_equal node(:blog).path, json['items'][0]['docUrl']
     assert_equal "Blog post",       json['items'][0]['docTitle']
     assert_equal 13,                json['items'][0]['docId']
-    
+
+    assert matcher =~ json
+
+  end
+
+  test 'search profiles functionality' do
+    get '/api/srch/profiles?srchString=Blog'
+    assert last_response.ok?
+
+    # Expected search pattern
+    pattern = {
+      srchParams: {
+        srchString: 'Blog',
+        seq: nil,
+      }.ignore_extra_keys!
+    }.ignore_extra_keys!
+
+    matcher = JsonExpressions::Matcher.new(pattern)
+
+    json = JSON.parse(last_response.body)
```

Comment:
Hi, here, we'd want to search for a profile, not "Blog" - so we could change `get '/api/srch/profiles?srchString=Blog'` to `get '/api/srch/profiles?srchString=Bob' (working from the test data in `/test/fixtures/users.yml`). 

Then, as done above on [these lines](https://github.com/publiclab/plots2/pull/1437/files#diff-ae2647e0aa24f1474d9192f6cedc811eR27), we can test the returned data. In this case, we'd want to check that the returned data is the record for "Bob". We could make an educated guess but maybe the best would be to start by inserting a logging action to see what the contents are:

```
puts json['items'][0]
```

Once we know exactly what it returns, we can adjust the test to confirm that. Does this make sense?